src/wasm_plugin.rs

```diff
@@ -24,12 +24,23 @@ impl PluginHandler<Configuration> for MarkdownPluginHandler {
         resolve_config(config, global_config)
     }
 
+    // Markdown extensions: markdown, mdown, mkdn, mdwn, mkd, md
+    // ref: https://superuser.com/questions/249436/file-extension-for-markdown-files/285878#285878
+    // ref: https://github.com/denoland/deno_registry2/issues/206
+
     fn get_plugin_info(&mut self) -> PluginInfo {
         PluginInfo {
             name: env!("CARGO_PKG_NAME").to_string(),
             version: env!("CARGO_PKG_VERSION").to_string(),
             config_key: "markdown".to_string(),
-            file_extensions: vec!["md".to_string()],
+            file_extensions: vec![
+                "md".to_string(),
+                "mkd".to_string(),
+                "mdwn".to_string(),
+                "mkdn".to_string(),
+                "mdown".to_string(),
+                "markdown".to_string(),
```

Comment:
Wow. I did not know about this. I think it's good to add these here, but I'm going to leave it out of the config file that ends up being created with `dprint init`. Luckily they are separate.

Reply:
Reasonable; `md` and `mkd` seem to be the most predominate suffixes.